coordinator/testing/src/main/java/io/stargate/it/cql/AbstractCompressionTest.java

```diff
@@ -40,6 +40,6 @@ protected void compressionTest(CqlSession session) {
     // We are testing with small responses, so the compressed payload is not even guaranteed to be
     // smaller.
     assertThat(executionInfo.getResponseSizeInBytes()).isGreaterThan(0);
-    assertThat(executionInfo.getCompressedResponseSizeInBytes()).isGreaterThan(0);
+    // assertThat(executionInfo.getCompressedResponseSizeInBytes()).isGreaterThan(0);
```

Comment:
?

Reply:
Driver is always returning the value of executionInfo.getCompressedResponseSizeInBytes() as -1 with protocol v5, similar to what is returned against cassandra.